CMakeLists.txt

```diff
@@ -206,9 +208,7 @@ maketestcpp(path-search)
 
 maketestcpp(performance)
 if(LIBLO_FOUND)
-    target_include_directories(performance PRIVATE ${LIBLO_INCLUDE_DIRS})
-    target_link_libraries(performance      PRIVATE ${LIBLO_LIBRARIES})
-    target_compile_definitions(performance PRIVATE HAVE_LIBLO)
```

Comment:
Why did you remove the `HAVE_LIBLO` define? Wouldn't this mean that `performance.cpp` compiles without liblo?

Reply:
Oops, mb. Should have noticed that one. Will fix now

Reply:
Forgot about this pr. Will get back to this soon.

Reply:
In my latest commit I pushed the performance test into liblo_found, this will do the same thing I guess. I'm not resolving this for now so as to keep my approach open for discussion.

Reply:
I misunderstood the comment. I fixed it now.